lib/matplotlib/sankey.py

```diff
@@ -525,7 +521,7 @@ def add(self, patchlabel='', flows=None, orientations=None, labels='',
             if orient == 1:
                 if is_input:
                     angles[i] = DOWN
-                elif not is_input:
+                elif is_input is False:
```

Comment:
```suggestion
                elif not is_input:
```

Reply:
If I remember correctly, the original code actually breaks for None (I did this a few months ago, but pushed to the main repo #22916 ). See the comment on the next line.

Recall that `elif not is_input:` is simply `else:` as `not None` is `True`.